src/content/docs/en/guides/content-collections.mdx

```diff
@@ -534,34 +534,34 @@ const { Content } = await entry.render();
 ---
 ```
 
-## Migrating from File-Based Routing
+## Переход от маршрутизации на основе файлов
 
-If you have an existing Astro project, such as a blog, that uses Markdown or MDX files in subfolders inside `src/pages/`, consider migrating related content or data files to content collections. 
+Если у вас есть существующий проект Astro, например блог, в котором используются файлы Markdown или MDX во вложенных папках внутри `src/pages/`, рассмотрите возможность переноса связанного контента или файлов данных в коллекции контента. 
  
-See how to convert a basic blog example from `src/pages/posts/` to `src/content/posts` in our [step-by-step tutorial](/en/tutorials/add-content-collections/) that uses the codebase from [the Build a Blog tutorial's finished project](https://github.com/withastro/blog-tutorial-demo).
+Посмотрите, как преобразовать базовый пример блога из `src/pages/posts/` в `src/content/posts` в нашем [пошаговом руководстве](/ru/tutorials/add-content-collections/), которое использует кодовую базу из [готового проекта руководства Build a Blog](https://github.com/withastro/blog-tutorial-demo).
 
```

Comment:
Refactored a bit, ensuring consistency in wording and translating the project name for coherence.
```suggestion
Посмотрите, как преобразовать базовый пример блога из `src/pages/posts/` в `src/content/posts` в нашем [пошаговом руководстве](/ru/tutorials/add-content-collections/), которое использует кодовую базу из [готового проекта учебного пособия Создать Блог](https://github.com/withastro/blog-tutorial-demo).
```